pkg/engine/validate/common.go

```diff
@@ -7,6 +7,13 @@ import (
 	"github.com/nirmata/kyverno/pkg/engine/operator"
 )
 
+type ValidationFailureReason int
```

Comment:
In reference to some previous comments, a new type is introduced but this doesn't really add much value.
As the enum is not `PathNotFound` and the other value is `RuleFailure`(which is already default behavior if the rule has failed).

Reply:
See above.